apps/core/scenes/search/Search.js

```diff
@@ -101,6 +112,7 @@ class Search extends React.Component<Props> {
           </View>
         </View>
         <SearchModal />
+        <Alert message="Please fill this form completely before you proceed" />
```

Comment:
I'm still not sure about this implementation. What if there will be multiple `Alert` components on one screen (or in nested components)?
Context doesn't seems to add much in this case.
I liked @tbergq original idea to have just one `Alert` in some higher component. And be able to just call something like `showAlert('message')` on alert context to display the message.

Reply:
Sure, I will change it.

Reply:
Thanks. For web we can probably add it directly into `_app.js` or `Layout`. I'm just not that sure about mobile, probably `App.js` can also work.